libp2p/utility.nim

```diff
@@ -35,3 +36,8 @@ func shortLog*(item: string): string =
     result &= item[0..<split]
     result &= "..."
     result &= item[(item.len - split)..item.high]
+
```

Comment:
we should not have this in the library, this should be setup and provided by the consumming application.

Reply:
I suppose you refer to the http server ?
Comment was pointing something else.

Anyway yes I will revert that if so, I was planning to remove it anyway, it was just to show to Stefan.

Reply:
yeah, missed it by one line, sorry and you can't edit it once submitted 🤷‍♂️

Reply:
hahah, no worries, I figured 😄 , didn't know you cannot edit tho